test/org/sagebionetworks/bridge/services/email/ParticipantRosterProviderTest.java

```diff
@@ -29,6 +29,20 @@ public void setUp() throws Exception {
     }
 
     @Test
+    public void correctlySplitsRecipients() throws Exception {
+        StudyParticipant participant = new StudyParticipant();
+        List<StudyParticipant> participants = Lists.newArrayList(participant);
+        
+        study.setConsentNotificationEmail("bridge-testing@sagebase.org,postmaster@sagebase.org");
+        ParticipantRosterProvider provider = new ParticipantRosterProvider(study, participants);
+        
+        MimeTypeEmail email = provider.getMimeTypeEmail();
+        assertEquals(2, email.getRecipientAddresses().size());
+        assertEquals("bridge-testing@sagebase.org", email.getRecipientAddresses().get(0));
+        assertEquals("postmaster@sagebase.org", email.getRecipientAddresses().get(1));
+    }
```

Comment:
Since commaListToSet creates a set, the order isn't guaranteed. You're better off either converting this to a set and comparing set equality or changing the util function to commaListToList (or whatever name makes more sense).